kotlin/show-callout/build.gradle

```
@@ -0,0 +1,27 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion rootProject.ext.compileSdkVersion
+
+    defaultConfig {
+        applicationId "com.esri.arcgisruntime.sample.showcallout"
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
+        versionCode rootProject.ext.versionCode
+        versionName rootProject.ext.versionName
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    // arcgis-android & appcompat lib dependencies from rootProject build.gradle
+    implementation "com.android.support.constraint:constraint-layout:$constraintLayoutVersion"
```

Are we supposed to use the androidx dependency here? I'm not sure, it seems that both work fine. I'll put this here anyway.

```suggestion
    implementation "androidx.constraintlayout:constraintlayout:$constraintLayoutVersion"
```

---

oh cool, thanks for pointing that out! I checked with Trevor, yeh you're right, we should: I've updated the wiki too!